src/applications/mhv-secure-messaging/components/Search/SearchForm.jsx

```diff
@@ -357,13 +377,27 @@ const SearchForm = props => {
               />
             )
           )}
-          {filtersCleared && (
+          {filterStatus === FilterStatus.DEFAULT && (
+            <span className="sr-only" aria-live="polite" id="filter-default">
+              No filters applied
+            </span>
+          )}
+          {filterStatus === FilterStatus.APPLIED && (
+            <span
+              className="sr-only"
+              aria-live="polite"
+              id="filter-applied-success"
+            >
+              Filters successfully applied
+            </span>
+          )}
+          {filterStatus === FilterStatus.CLEARED && (
             <span
               className="sr-only"
               aria-live="polite"
               id="filter-clear-success"
             >
-              Filters succesfully cleared
+              Filters successfully cleared
             </span>
           )}
```

Comment:
suggestion - this repetitive code may be simplified. but not too big of a deal, just a preference
```suggestion
const FilterStatus = {
  DEFAULT: 'filter-default',
  APPLIED: 'filter-applied-success',
  CLEARED: 'filter-clear-success',
};

          <span className="sr-only" aria-live="polite" id={filterStatus}>
            {filterStatus === FilterStatus.DEFAULT && 'No filters applied'}
            {filterStatus === FilterStatus.APPLIED &&
              'Filters successfully applied'}
            {filterStatus === FilterStatus.CLEARED &&
              'Filters successfully cleared'}
          </span>
```